app/test/admin/api_actions_test.dart

```diff
@@ -155,28 +155,4 @@ void main() {
         .uploadPackageBytes(bytes);
     expect(rs2.success.message, contains('Successfully uploaded'));
   });
-
-  testWithProfile('setting and viewing secrets', fn: () async {
```

Comment:
Not sure: what was wrong with this test?

Reply:
You can set secrets when we are using the secret manager, the action to interact with secrets is gone.